src/components/scenes/RequestScene.js

```diff
@@ -386,11 +417,11 @@ export class Request extends Component<Props, State> {
     if (this.amounts) {
       const native = parseFloat(this.amounts.nativeAmount)
```

Comment:
remove const declration and se bns.lte(native, '0') in if

Reply:
done